pallets/automation-time/src/migrations.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{Config, Weight};
+use frame_support::traits::Get;
+use log::info;
+
+pub mod v1 {
+	use frame_support::{migration::get_storage_value};
+
+  use crate::{LastTimeSlot, AutomationTimeStorageVersion};
+
+use super::*;
+
+	pub fn migrate<T: Config>() -> Weight {
+		info!(target: "automation-time", "Migrating automation-time v1");
+    let pallet_prefix: &[u8] = b"AutomationTime";
+		let storage_item_prefix: &[u8] = b"LastTimeSlot";
+
+		let stored_data = get_storage_value::<u64>(
+			pallet_prefix,
+			storage_item_prefix,
+      &[],
+		).expect("Must have last slot value");
+
+    // TODO: (jzhou) Gate migration on not having storage version
+    // TODO: (jzhou) What if stored_data is None?
```

Comment:
2 concerns:
1. what is last_time_slot is none for some reason? (like on first time deploy) this is less of a concern, i think we just skip the migration?
2. how do we make sure this doesn't run twice? i tried with storage_version as a new storage variable. but it'll be not set on a brand new chain too. in which case, we prob don't wanna run either. question is: how do we set storage version on a new chain so that it doesn't try to do this migration later down the road?